File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ClassicChair.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ClassicChair : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a classic chair");
        }
    }
}

File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ClassicChair.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ClassicChair implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a classic chair\n";
    }
}